saleor/core/utils/random_data.py

```diff
@@ -203,6 +205,9 @@ def create_product_variants(variants_data):
             continue
         defaults['product_id'] = product_id
         defaults['attributes'] = json.loads(defaults['attributes'])
+        if defaults['price_override']:
```

Comment:
Do you mind adding same logic for the `cost_price` field?

Reply:
i missed that - will do